src/main/java/io/botcrafting/botcraft/core/handler/chain/MessageChain.java

```diff
@@ -0,0 +1,27 @@
+package io.botcrafting.botcraft.core.handler.chain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import io.botcrafting.botcraft.core.handler.processors.MessageProcessor;
+import io.botcrafting.botcraft.core.model.Message;
+
+@Component
+public class MessageChain {
```

Comment:
MessageChain gives me the idea of a sequence of messages and not a sequence of operations being applied to Messages, maybe MessageSomeVerbChain would state the meaning of the action being performed by this chain of commands. Suggestion: MessageReplierChain